src/polyfills.js

```
@@ -0,0 +1,109 @@
+    return evt;
+  }
+  
+  window.CustomEvent = CustomEvent;
+})();
+
+// requestAnimationFrame polyfill
+(function() {
+  var lastTime = 0;
+  var vendors = ['ms', 'moz', 'webkit', 'o'];
+  for(var x = 0; x < vendors.length && !window.requestAnimationFrame; ++x) {
+    window.requestAnimationFrame = window[vendors[x]+'RequestAnimationFrame'];
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)